core/SettingsPiwik.php

```diff
@@ -427,6 +427,10 @@ public static function overwritePiwikUrl($currentUrl)
      */
     public static function isHttpsForced()
     {
+        if (!SettingsPiwik::isPiwikInstalled()) {
```

Comment:
Just curious why we check for this? Wouldn't it use pretty much the default value from global ini anyway? And would it be bad to redirect during installation?